controllers/spiaccesstokendataupdate_controller.go

```diff
@@ -96,6 +96,8 @@ func (r *SPIAccessTokenDataUpdateReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, fmt.Errorf("failed to delete the processed data token update: %w", err)
 	}
 
+	lg.V(logs.DebugLevel).Info(fmt.Sprintf("token data update deleted: %s", req.Name))
```

Comment:
do we want to keep it here ?

Reply:
I'd prefer to have this debug info, can remove if critical for you

Reply:
ok, I'm fine with that